test/com/facebook/buck/rules/TargetGraphTest.java

```diff
@@ -141,8 +141,7 @@ public void getAllReportsMissingNode() {
     expectedException.expectMessage(
         "Required target for rule '//foo:bar#baz' was not found in the target graph.");
     // Force the Iterable to evaluate and throw.
-    ImmutableSet.copyOf(
-        targetGraph.getAll(ImmutableSet.of(BuildTargetFactory.newInstance("//foo:bar#baz"))));
+    targetGraph.getAll(ImmutableSet.of(BuildTargetFactory.newInstance("//foo:bar#baz")));
```

Comment:
This change actively breaks this test because targetGraph.getAll is lazy. I have just sent out a change to instead iterate over the values and toString() them. Please revert this bit of the change.